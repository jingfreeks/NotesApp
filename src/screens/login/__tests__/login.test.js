import React from 'react';
import { renderWithProviders } from "../../../utils/testframework";
import LoginScreen from '../login';


describe('Login Screen', () => {
  it('Should work as expected to get snapshot', () => {
    const all = renderWithProviders(<LoginScreen />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
