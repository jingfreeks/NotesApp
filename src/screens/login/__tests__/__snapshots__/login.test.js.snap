// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Screen Should work as expected to get snapshot 1`] = `
<View
  onLayout={[Function]}
  style={
    [
      [
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ],
      {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      [
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "marginLeft": 10,
          "marginRight": 10,
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "alignItems": "center",
            "justifyContent": "center",
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#000000",
              "fontFamily": "Avenir",
              "fontSize": 24,
              "fontWeight": "900",
            },
          ]
        }
      >
        LOG IN
      </Text>
    </View>
    <View
      style={
        [
          {},
        ]
      }
    >
      <RCTScrollView
        style={
          [
            {
              "marginBottom": 10,
            },
          ]
        }
      >
        <View>
          <View
            style={
              [
                {
                  "paddingBottom": 10,
                  "paddingLeft": 10,
                  "paddingRight": 10,
                  "paddingTop": 10,
                },
              ]
            }
          >
            <View
              style={
                [
                  {
                    "flexDirection": "column",
                  },
                ]
              }
            >
              <Text
                style={
                  [
                    {
                      "color": "#0e3f6c",
                      "fontFamily": "Avenir",
                      "fontSize": 18,
                      "fontWeight": "900",
                    },
                  ]
                }
              >
                Email
              </Text>
              <View
                style={
                  [
                    {
                      "flexDirection": "row",
                    },
                  ]
                }
              >
                <TextInput
                  autoCapitalize="none"
                  onBlur={[Function]}
                  onChangeText={[Function]}
                  placeholder="Email"
                  secureTextEntry={false}
                  style={
                    [
                      {
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "height": 30,
                        "marginBottom": 5,
                      },
                    ]
                  }
                  value="lyndell.dobluis@gmail.com"
                />
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "#C2D6DC",
                        "fontSize": 25,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
              <View
                style={
                  [
                    {
                      "alignSelf": "stretch",
                      "borderColor": "#C2D6DC",
                      "borderWidth": 0.5,
                      "flexDirection": "row",
                      "marginBottom": 15,
                    },
                  ]
                }
              />
            </View>
          </View>
          <View
            style={
              [
                {
                  "paddingBottom": 10,
                  "paddingLeft": 10,
                  "paddingRight": 10,
                  "paddingTop": 10,
                },
              ]
            }
          >
            <View
              style={
                [
                  {
                    "flexDirection": "column",
                  },
                ]
              }
            >
              <Text
                style={
                  [
                    {
                      "color": "#0e3f6c",
                      "fontFamily": "Avenir",
                      "fontSize": 18,
                      "fontWeight": "900",
                    },
                  ]
                }
              >
                Password
              </Text>
              <View
                style={
                  [
                    {
                      "flexDirection": "row",
                    },
                  ]
                }
              >
                <TextInput
                  autoCapitalize="none"
                  onBlur={[Function]}
                  onChangeText={[Function]}
                  placeholder="Password"
                  secureTextEntry={true}
                  style={
                    [
                      {
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "height": 30,
                        "marginBottom": 5,
                      },
                    ]
                  }
                  value="Jingfreeks99"
                />
                <Text
                  allowFontScaling={false}
                  onPress={[Function]}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "#C2D6DC",
                        "fontSize": 25,
                      },
                      undefined,
                      {
                        "fontFamily": "Material Icons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
              <View
                style={
                  [
                    {
                      "alignSelf": "stretch",
                      "borderColor": "#C2D6DC",
                      "borderWidth": 0.5,
                      "flexDirection": "row",
                      "marginBottom": 15,
                    },
                  ]
                }
              />
            </View>
          </View>
        </View>
      </RCTScrollView>
      <View
        style={
          [
            {
              "borderColor": "grey",
              "borderRadius": 10,
              "borderWidth": 1,
            },
          ]
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderRadius": 1,
              "opacity": 1,
              "paddingBottom": 15,
              "paddingLeft": 15,
              "paddingRight": 15,
              "paddingTop": 15,
            }
          }
        >
          <Text
            style={
              [
                {
                  "color": "#0e3f6c",
                  "fontFamily": "Avenir",
                  "fontSize": 16,
                  "fontWeight": "400",
                },
              ]
            }
          >
            Login
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
